.travis.yml

```diff
@@ -27,7 +27,7 @@ matrix:
       scala: 2.11.7
       env: TEST_SPARK_VERSION="1.5.0"
 script:
-  - sbt -Dspark.testVersion=$TEST_SPARK_VERSION ++$TRAVIS_SCALA_VERSION coverage test
```

Comment:
Sure. Let me go ahead and merge this now. FYI, we should use the `./dev/merge_pr.py` script to merge pull requests and _not_ the GitHub merge button (similar to what we do in our other repos and in Spark itself).